requirements.txt

```diff
@@ -1 +1,2 @@
 requests
+
```

Comment:
Ideally this spurious change would be excluded. (I'm guessing it's just left over from experimenting with `toolz` or similar.)